corva/application.py

```diff
@@ -0,0 +1,52 @@
+from typing import Callable, List, Optional
+
+from corva.middleware.unpack_context import unpack_context
+from corva.models.base import BaseContext
+from corva.types import MIDDLEWARE_CALL_TYPE, MIDDLEWARE_TYPE
+
+
+def wrap_call_in_middleware(
+     call: Callable,
+     middleware: Optional[List[MIDDLEWARE_TYPE]] = None
+) -> MIDDLEWARE_CALL_TYPE:
```

Comment:
what is the difference between `Callable` and `MIDDLEWARE_CALL_TYPE` types?

Reply:
`Callable` doesn't specify arguments, that callable accepts. whereas `MIDDLEWARE_CALL_TYPE` provides them